packages/fuels-abigen-macro/tests/harness.rs

```diff
@@ -1342,6 +1344,7 @@ async fn test_array() {
 }
 
 #[tokio::test]
+#[ignore]
```

Comment:
Why is this one ignored?

Reply:
#409 will fix this and we will delete ``ignore``

Reply:
Since https://github.com/FuelLabs/fuels-rs/pull/409 has been merged, I think we can remove this `#[ignore]` from here.

Reply:
Deleted!